guardduty-malware-s3/README.md

```diff
@@ -0,0 +1,89 @@
+# AWS S3 to AWS GuardDuty to EventBridge to SNS
+
+This pattern creates a new Malware Protection plan for the protected resource. Presently, S3Bucket is the only supported protected resource. The SAM template creates an IAM role that includes the permissions required to scan and (optionally) add tags to the associated protected resource. The template also creates an EventBridge managed rule, which is used to route S3 put object and S3 bucket-level event notifications to GuardDuty's S3 Malware Protection Service. Another EventBridge rule, created by the template, monitors the S3 object scan results and sends notifications to the user’s email address, which is subscribed to an Amazon SNS topic.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/guardduty-malware-s3
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd guardduty-malware-s3
+    ```
+1. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+    ```
+    sam deploy --guided
+    ```
+1. During the prompts:
+    * Enter a stack name.
+    * Enter the desired AWS Region.
+    * Enter the name of S3 bucket to be protected.
+    * Enter ENABLED or DISABLED for the option to add tags to the S3 objects after the scan ends.
+    * Enter an email address to receive the GuardDuty Malware Protection Object Scan Results.
+    * Allow the SAM CLI to create IAM roles with the required permissions.
+
+    Once you have run `sam deploy --guided` mode once and saved arguments to a configuration file (samconfig.toml), you can use `sam deploy` in future to use these defaults.
+
+1. Note the outputs from the SAM deployment process. These contain the resource names and/or ARNs which are used for testing.
+
```

Comment:
Add a step to confirm the email subscription.

Reply:
Thanks @ellisms. I have added a step to confirm the email subscription as suggested.